src/components/NewsletterSignup/NewsletterSignup.stories.tsx

```diff
@@ -139,7 +127,40 @@ export const WithControls: Story = {
     },
     jest: "NewsletterSignup.test.tsx",
   },
-  render: (args) => <NewsletterSignup {...args} />,
+  render: ({ title, descriptionText, errorText, ...rest }) => {
+    // Workaround for Storybook title control
```

Comment:
This is because the props can take in string or JSX.Element values? This makes sense and not for this PR but there are other components with similar prop value issues. Something like this would be nice in a util file and function so other Stories can reuse this (but again, you don't need to do this for this PR).

Normally, for these type of cases we turn off the control for the prop and give explicit values of all possible input values in the docs page. Just curious, when I do pass in HTML values for these props, it doesn't seem to work. What have you tried to pass that works?

Reply:
Removed the code. Disabled the control for the title prop.